src/host/ft_host/API_OutputTests.cpp

```diff
@@ -363,12 +364,6 @@ void OutputTests::WriteConsoleOutputWNegativePositions()
             VERIFY_ARE_EQUAL(expectedItem, readItem);
         }
     }
-
-    // Set the region so the left will end up past the right
-    adjustedRegion = region;
-    adjustedRegion.Left = -(adjustedRegion.Right + 1);
-    affected = adjustedRegion;
-    VERIFY_WIN32_BOOL_FAILED(WriteConsoleOutputW(consoleOutputHandle, buffer.data(), regionDimensions, regionOrigin, &affected));

```

Comment:
This test also relied on a bug:
https://github.com/microsoft/terminal/blob/ec9289288e6581b8d1cbb5f7d334422de4dee07a/src/host/directio.cpp#L486-L488

Check out the `<` operator! It's the wrong way around. :)

So, what this test actually asserted on is that we return `E_INVALIDARG` if the write region is larger than the target region (which is actually completely fine). I could've fixed the test, but it doesn't actually do what it intended anyway:
> Set the region so the left will end up past the right

But the code sets `Left` to a negative value... It's not actually greater than `Right`.